### DISCUSSION TITLE
[Curriculum] As a teacher logged in to Terrastories, I can create a Curriculum

### DISCUSSION THREAD

**Discussion by FeminismIsAwesome** [Terrastories/terrastories#732] [Temporarily Archived Tickets]
Larger Context:
Let’s create a Curriculum Builder feature in Terrastories! The idea behind a Curriculum Builder is to give users the ability to curate a selection of stories that will play in a certain order.

Let’s say there are 500 stories in Terrastories for a community. This is the entire archive of this community’s oral histories. But today, a teacher wants to focus on just a few of them that are thematically related, and play them a sequential order for their classroom.

For this story:
Let's focus on just the ability to have a teacher log in to administrate and go to a new Curriculum page, and make a Curriculum with a title and a description. 
![image](https://user-images.githubusercontent.com/5641692/78454597-dc5f1680-764d-11ea-8ee0-27e413206787.png)

**Comment by FeminismIsAwesome** [comment]
@rudokemper can you see if recent commit merges by sarah's work do this well enough to accept this issue or if we need more work for a first pass?

**Comment by rudokemper** [comment]
@FeminismIsAwesome @slaloggia I was able to do exactly what this ticket asks for! 👍 LGTM
